server/src/main/java/org/opensearch/index/translog/TranslogWriter.java

```diff
@@ -359,7 +359,7 @@ synchronized boolean assertNoSeqAbove(long belowTerm, long aboveSeqNo) {
      *
      * Note: any exception during the sync process will be interpreted as a tragic exception and the writer will be closed before
      * raising the exception.
-     * @return
+     * @return something
```

Comment:
This isn't very descriptive :-)

Reply:
Whoops, didn't mean to include that in the PR.  gradle was throwing an error because another PR didn't include javadocs properly so I did a temporary fix for my local